meta-resin-common/conf/distro/include/resin-os.inc

```diff
@@ -7,6 +7,11 @@ DISTRO_VERSION = "1.1.1"
 
 MAINTAINER = "Resin <hello@resin.io>"
 
+# Strip down unneeded features
+POKY_DEFAULT_DISTRO_FEATURES_remove = "ptest"
```

Comment:
Indeed. We try to unify the staging and production images. And then we will see how are we going to do automated testing.